src/_guides/language/sound-problems.md

```diff
@@ -466,29 +486,48 @@ filterValues((x) => (x [!as String!]).contains('Hello'));
 
 <hr>
 
-<a id="common-errors-and-warnings"></a>
-## Runtime errors
+### Incorrect type inference
 
-Dart enforces a sound type system. Roughly, this means
-you can't get into a situation where the value stored in a variable is
-different from the variable's static type. Like most modern statically
-typed languages, Dart accomplishes this with a combination of static
-(compile-time) and dynamic (runtime) checking.
+```nocode
+error - ...
+```
 
-For example, the following type error is detected at compile-time:
+On rare occasions, Dart's enhaced type inference may infer the wrong type for function literal arguments in a generic constructor invocation.
```

Comment:
```suggestion
On rare occasions, Dart's enhanced type inference might infer
the wrong type for function literal arguments in a generic constructor invocation.
```

Line length!!!  :D. We don't break at exactly 80 chars, we try to break on phrasing.

Reply:
https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks